Makefile

```diff
@@ -70,7 +70,11 @@ JUNIT_REPORT ?= /usr/local/bin/go-junit-report
 
 # Namespace and environment running the control plane.
 # A cluster must support multiple control plane versions.
-ISTIO_NS ?= istio-control
+ISTIO_SYSTEM_NS ?= istio-system
```

Comment:
I don't think the tests will pass... 

Still trying to get circleci back to working, to verify. 
How about: set telemetry/policy/gateway to istio-control, and add separate test pipeline for having them in separate ns ?

Reply:
Yes, let me try that locally.